src/raul/card

```diff
@@ -0,0 +1,67 @@
+public class Main {
+    public static void main(String[] args){
+
```

Comment:
In order to make clean code, empty lines should be consistent with the rest of the code. Ideally, there should not be an empty line between method signature and method's code.